src/drivers/actuators/vertiq_io/module.yaml

```diff
@@ -0,0 +1,271 @@
+module_name: Vertiq IO
+serial_config:
+    - command: vertiq_io telemetry ${SERIAL_DEV}
+      port_config_param:
+        name: VERTIQ_IO_CFG
+        group: Vertiq IO
+
+actuator_output:
+  output_groups:
+    - param_prefix: VERTIQ_IO
+      group_label: 'ESCs'
+      channel_label: 'ESC'
+      standard_params:
+        min: { min: 0, max: 65535, default: 0 }
+        max: { min: 0, max: 65535, default: 65535 }
+        failsafe: { min: 0, max: 500 }
+      num_channels: 16
+
+parameters:
+    - group: Vertiq IO
+      definitions:
+        VERTIQ_BAUD:
+            description:
+                short: The IQUART driver's baud rate
+                long: |
+                    The baud rate (in bits per second) used by the serial port connected with IQUART communication
+            type: int32
+            default: 115200
+            reboot_required: true
+        VERTIQ_NUM_CVS:
+            description:
+              short: The number of Vertiq IFCI parameters to use
+              long: |
+                  The total number of IFCI control variables being used across all connected modules
+            type: int32
+            default: 0
+            min: 0
+            max: 16
+            reboot_required: true
+        DISARM_TRIGGER:
+            description:
+                short: The triggered behavior sent to the motors on PX4 disarm
+                long: |
+                    The behavior triggered when the flight controller disarms. You have the option to trigger your motors' disarm behaviors, set all motors to coast,
+                    or set a predefined throttle setpoint
+            type: enum
+            values:
+                0: Send Explicit Disarm
+                1: Coast Motors
+                2: Set Predefined Velocity Setpoint
+            default: 0
+        DISARM_VELO:
+            description:
+                short: The velocity value sent to the motors when DISARM_TRIGGER is set to Set Predefined Setpoint
+                long: This is the velocity that will be sent to all motors when PX4 is disarmed and DISARM_TRIGGER is Set Predefined Setpoint
+            type: int32
+            default: 0
+            min: 0
+            max: 100
+        ARMING_BEHAVE:
+            description:
+                short: The triggered behavior on PX4 arm
+                long: |
+                    The behavior triggered when the flight controller arms. You have the option to use your motors' arming behaviors, or to force all of your motors to arm
+            type: enum
+            values:
+                0: Use Motor Arm Behavior
+                1: Send Explicit Arm Command
+            default: 0
+        TARGET_MODULE_ID:
+            description:
+                short: The Module ID of the module you would like to communicate with
+                long: |
+                    This is the value used as Object ID to instantiate the Vertiq C++ API objects in the project.
```

Comment:
This description may be a bit misleading at this point. It isn't used for all the Vertiq API objects, just the configuration ones. If I make an operational one with a fixed module ID, this won't change it I believe